docs/rules/no-unused-expressions.md

```diff
@@ -59,6 +59,8 @@ delete a.b
 void a
 ```
 
+Strict mode directives are also not considered problems.
```

Comment:
I don't agree with this addition. First of all, *all* directives are not problems, and this should be obvious because directives are not expressions and this rule only ever reports expressions. Are we going to say that `if` statements are not problems? Function declarations? Import/export declarations? `do`-`while` statements? Lexical declarations? No, none of these need to be individually called out.